README.md

```diff
@@ -114,8 +117,11 @@ The story of the project is pretty interesting. You are a famous magician in the
       <div class = "column">
         <img src = "assets/Screenshots/Flying Wizard/ss23.png" height = "200px">
       </div>
-  </div>
+  </div> -->
+
 
+<img src = "assets/Screenshots/gif/Flying-w/Flying-wizard.gif
```

Comment:
This links seems to be broken, Could you please fix it.
[Ref:]
<img src="https://i.imgur.com/GkdUfdd.png">